packages/tables/src/styled/StyledCell.ts

```diff
@@ -16,7 +16,7 @@ const COMPONENT_ID = 'tables.cell';
 
 export interface IStyledCellProps
   extends Pick<ICellProps, 'isMinimum' | 'isTruncated' | 'hasOverflow' | 'width'> {
-  size?: ITableProps['size'];
+  $size?: ITableProps['size'];
```

Comment:
Please note that the picked props above will continue to leak in `styled-components` v6.

Reply:
I went ahead and converted the remaining props to transient props and I _think_ captured all the remaining spreadable props as explicitly converted. Doing another self review now, as well.